lib/import-export-visitor.js

```diff
@@ -468,6 +525,8 @@ module.exports = class ImportExportVisitor extends Visitor {
 
         path.replace(this._buildExportDefaultStatement(dd));
       }
+
+      this.madeChanges = true;
```

Comment:
Could the `enforceStrictMode` property be re-used for this (as to not introduce another prop)?